packages/typescript/index.js

```diff
@@ -55,6 +55,86 @@ module.exports = {
       "allow-pascal-case",
       "ban-keywords",
       "check-format"
-    ]
+    ],
+    "arrow-return-shorthand": true,
+    "callable-types": true,
+    "class-name": true,
+    "comment-format": [
+      true,
+      "check-space"
+    ],
+    "curly": true,
+    "eofline": true,
+    "forin": true,
+    "import-blacklist": [
+      true,
+      "rxjs"
+    ],
+    "import-spacing": true,
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "interface-over-type-literal": true,
+    "label-position": true,
+    "max-line-length": [
+      true,
+      140
```

Comment:
I believe 140 is the default. rooms uses 100. we have to decide about that.

Reply:
We are using 100 as well.